src/main/scala/io/iohk/ethereum/network/ForkResolver.scala

```diff
@@ -17,17 +17,17 @@ object ForkResolver {
 
   class EtcForkResolver(blockchainConfig: BlockchainConfig) extends ForkResolver {
```

Comment:
Should we also rename this resolver to `EthereumForkResolver` maybe?
There's also various classes with an `Etc-` prefix (as the `EtcHandshaker`), should we rename them as well? We could create a separate task to do all this renaming.